technical-docs/2021-05-remote-restore-redesign.md

```diff
@@ -0,0 +1,110 @@
+# Medusa Operator - Restore redesign document
+
+## Background
+
+Medusa operator doesn’t yet support remote restores: restoring a backup to another cluster.
+As we implement this feature, we would like to have it support multiple use cases that operators could expect to have working:
+
+- Restoring a medusa-operator based backup to another Cassandra install in the same k8s cluster
+![](img/medusa_k8s-new-restore-design-1.jpg)
+
+- Restoring a medusa-operator based backup to another Cassandra install in a different k8s cluster
+![](img/medusa_k8s-new-restore-design-2.jpg)
+
+- Restoring a Medusa backup (non k8s) to a k8s Cassandra install
+![](img/medusa_k8s-new-restore-design-3.jpg)
+
+
+
+
+## Sync CassandraBackup resources with storage
```

Comment:
Yes, but they would use different prefixes assuming multitenancy is turned on. That's why I'd want to be able to override the prefix at sync/restore time.
It would be up to the ops to change the storage bucket settings after a restore if needed.

Reply:
Ok cool, so they wouldn't be storing backups in the same location. 👍